test/integration/crud/crud.prose.test.js

```diff
@@ -0,0 +1,43 @@
+const { expect } = require('chai');
+const { MongoBulkWriteError } = require('../../../src');
+
+describe('CRUD Prose Spec Tests', () => {
+  let client;
+
+  beforeEach(async function () {
+    client = this.configuration.newClient({ monitorCommands: true });
+    await client.connect();
+  });
+
+  afterEach(async () => {
+    if (client) {
+      await client.close();
+    }
+  });
+
+  // Note: This test does not fully implement the described test, it's missing command monitoring assertions
```

Comment:
Not adding a TODO ticket here because this is just getting a jump start on the spec compliance analysis work that will be done in NODE-3719, I only moved it at all because I knew it was intended to be a spec test (since I wrote it)